Readme.md

```diff
@@ -1,11 +1,19 @@
 # USB boot code
 
 This is the USB MSD boot code which should work on the Raspberry Pi model A, Compute Module, Compute
-module 3 and Raspberry Pi Zero.
+module 3, Compute Module 4 and Raspberry Pi Zero.
```

Comment:
It's not your typo, but could you capitalise "module"?

Reply:
done